index.js

```diff
@@ -64,11 +64,8 @@ var uncaughtException = require('./lib/interfaces/uncaught.js');
  */
 function initializeClientAndInterfaces ( initConfiguration ) {
 
-  var config = gatherConfiguration(initConfiguration);
-  var client = new AuthClient(
-    config.projectId,
-    config.shouldReportErrorsToAPI
-  );
+  var config = new Configuration(initConfiguration).init();
```

Comment:
@ofrobots It could be, but then this disallows testing the instances default state when it is initialized and the return values of individual state reduction functions on a yet to be inited instance. I feel that these are valuable tests for determining how a 'un-inited' instance behaves which may occur if there is no user and environmental configuration.